src/lib/okta/connect.ts

```diff
@@ -0,0 +1,16 @@
+import OktaJwtVerifier from "@okta/jwt-verifier";
+import { AUTH_PROTOCOL, AUTH_URL, AUTH_SERVER, SECURE } from "../../config";
+
+const oktaJwtVerifier = new OktaJwtVerifier({
+    issuer: `${AUTH_PROTOCOL}://${AUTH_URL}/oauth2/${AUTH_SERVER}`,
+    assertClaims: {
+        aud: `api://${AUTH_SERVER}`
+    },
+    testing: {
```

Comment:
As of:
[https://github.com/okta/okta-oidc-js/blob/40fe1dad11df4178eff20f6edd268877fb964ce7/packages/jwt-verifier/test/integration-test/resource-server.js#L26](Okta JWT Verifier)